src/lib/addons.js

```diff
@@ -46,10 +48,17 @@ const initSwagger = (app) => {
   app.use(swagger.getMiddleware(swaggConfig));
 };
 
+const initHydration = async () => {
+  await hydrate();
+  const hydration = new CronJob('0 0 * * * *', hydrate, undefined, true);
```

Comment:
Cron notation is cool but we sometimes forget what it actually means
```suggestion
  // Run every hour
  const hydration = new CronJob('0 0 * * * *', hydrate, undefined, true);
```

Reply:
i don't entirely agree, but i'm happy to do more docs